docs/JLBP-0018.md

```diff
@@ -22,7 +22,8 @@ There are a number of problems with shading:
   to also roll in the security fix).
 - Shading cannot be performed on types used in a library's own public API as return
   types or method arguments.
-- Service entries under `META-INF/services` can easily be messed up. They need
+- Shading often corrupts non-code metadata in META-INF/MANIFEST.mf and other locations.
+  For instance, service entries under `META-INF/services` can easily need
   special effort to be merged because the entries are located in the same place
```

Comment:
"can easily need special effort to be merged" is difficult to read. Can you make it active voice?

Reply:
improved